src/Runtime/ObjectProcessors.cs

```diff
@@ -644,41 +619,44 @@ private async Task<EnterpriseCA> ProcessEnterpriseCA(ISearchResultEntry entry, R
                 ret.EnabledCertTemplates = _certAbuseProcessor.ProcessCertTemplates(entry.GetArrayProperty(LDAPProperties.CertificateTemplates), resolvedSearchResult.Domain).ToArray();
             }
 
-            if ((_methods & ResolvedCollectionMethod.Container) != 0)
+            if ((_methods & ResolvedCollectionMethod.Container) != 0 || (_methods & ResolvedCollectionMethod.CertServices) != 0)
             {
                 ret.ContainedBy = _containerProcessor.GetContainingObject(entry.DistinguishedName);
             }
 
-            // Collect properties from CA server registry
-            bool cASecurityCollected = false;
-            bool enrollmentAgentRestrictionsCollected = false;
-            bool isUserSpecifiesSanEnabledCollected = false;
-            var caName = entry.GetProperty(LDAPProperties.Name);
-            var dnsHostName = entry.GetProperty(LDAPProperties.DNSHostName);
-            if ((_methods & ResolvedCollectionMethod.CARegistry) != 0 && caName != null && dnsHostName != null)
+            if ((_methods & ResolvedCollectionMethod.CARegistry) != 0)
             {
-                ret.HostingComputer = await _context.LDAPUtils.ResolveHostToSid(dnsHostName, resolvedSearchResult.Domain);
-
-                CARegistryData cARegistryData = new()
+                 // Collect properties from CA server registry
+                var cASecurityCollected = false;
+                var enrollmentAgentRestrictionsCollected = false;
+                var isUserSpecifiesSanEnabledCollected = false;
```

Comment:
Just curious why we changed these from explicit to implicit bool? Is it a style thing we should observe elsewhere?

Reply:
We use implicit vars everywhere else, so style wise this is correct

Reply:
Will keep that in mind when contributing in the future, thanks for the clarification